### PR TITLE
Integrate Google maps api 

### PR DESCRIPTION
**Pull request by Karyum**
#87

### CODE REVIEW

File: views/event_details.hbs [hunk 1]
```diff
@@ -46,3 +46,21 @@
   </div>
 
 </div>
+
+<script type="text/javascript">
+  function initMap() {
```
Comment (finnhodgkin) [inline views/event_details.hbs]: Why is this inline but everything else in script files?

File: tests/tests.js [hunk 2]
```diff
@@ -162,13 +168,26 @@ tape('Test the event details route', async t => {
...
     .get('/en/events/someid')
     .end((err, res) => {
       t.error(err, 'event details /:lang/events/:id did not return an error');
+    });
+
+  nock(process.env.URI)
+    .get('/events/2')
+    .reply(200, {
+      en: { name: 'FAC' },
+      place: { en: { name: 'somehwere' }, location: [32.2323, 33.23232] }
```
Comment (finnhodgkin) [inline tests/tests.js]: somewhere*

File: tests/tests.js [hunk 3]
```diff
@@ -150,7 +150,13 @@ tape('Test the event details route', async t => {
     .get('/events/1')
     .reply(200, {
       en: { name: 'FAC' },
-      place: { en: { name: 'somehwere' } }
+      place: { en: { name: 'somehwere', address: 'Nazareth' } }
```
Comment (finnhodgkin) [inline tests/tests.js]: somewhere*

File: views/event_details.hbs (hunk 1 above)
Comment (Karyum) [reply views/event_details.hbs]: so this is here for 2 reasons : 
1) if it was on frontend i can't pass the lat & lng for the map ( unless if i maybe do an api request )
2) can't be on backend cause of i'm manipulating the html

Comment (finnhodgkin) [reply views/event_details.hbs]: Sure that'll do for me.

**Pull request merged**